uui/components/types.ts

```diff
@@ -26,6 +26,11 @@ export const allEpamBadgeSemanticColors: EpamBadgeSemanticColor[] = [
     'info', 'success', 'warning', 'critical', 'neutral',
 ];
 
+export type EpamTagSemanticColor = 'info' | 'success' | 'warning' | 'critical' | 'neutral';
```

Comment:
let's use `SemanticColor` type and just add 'neutral':
`export type EpamTagSemanticColor = SemanticColor | 'neutral';`
also let's fix it for `EpamBadgeSemanticColor`.

Also, i guess we don't need variables like  `allEpamTagSemanticColors`, since now we generate PE base on prop type. I guess we use them only for the PE configuration

Reply:
fixed